library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3287,4 +3286,31 @@
     <string name="onboarding_new_app_layout_feedback_message">Tap top right to see the option to feedback.</string>
     <string name="onboarding_new_app_layout_button_try">Try it out</string>
 
+    <string name="device_manager_filter_bottom_sheet_title">Filter</string>
+    <string name="device_manager_filter_option_all_sessions">All session</string>
```

Comment:
Maybe there is a missing ending "s" here?

Reply:
Right, done.